### PR TITLE
Library upgrades

### PR DESCRIPTION
**Pull request by mcarolan**

### CODE REVIEW

File: test/uk/gov/hmrc/fileupload/infrastructure/PlayHttpSpec.scala [hunk 1]
```diff
@@ -1,132 +0,0 @@
...
-import uk.gov.hmrc.play.audit.http.config.{AuditingConfig, BaseUri, Consumer, LoadAuditingConfig}
-import uk.gov.hmrc.play.audit.http.connector.AuditConnector
-import uk.gov.hmrc.play.config.RunMode
-import uk.gov.hmrc.play.http.HeaderCarrier
-import uk.gov.hmrc.play.test.UnitSpec
-
-import scala.collection.mutable.ListBuffer
-import scala.concurrent.ExecutionContext.Implicits.global
-
-class PlayHttpSpec extends UnitSpec with BeforeAndAfterEach with TestApplicationComponents with Eventually with FakeAuditer {
```
Comment (kastoestoramadus) [inline test/uk/gov/hmrc/fileupload/infrastructure/PlayHttpSpec.scala]: Why is it deleted?

Comment (mcarolan) [reply test/uk/gov/hmrc/fileupload/infrastructure/PlayHttpSpec.scala]: Removed it as the only thing it covers is auditing, and we don't use auditing (it's disabled in all environments)

Comment (YuanZHANG910) [reply test/uk/gov/hmrc/fileupload/infrastructure/PlayHttpSpec.scala]: Just in case you not have noticed that: This branch has conflicts with master now

**Comment by mcarolan** [comment]
Will open up a new pull request with a fixed version of PlayHttpSpec (you're right, no reason to believe auditing does not work)

**Pull request closed without merging**